ITTN-071.tex

```diff
@@ -15,19 +15,20 @@
 % \setDocSubtitle{A subtitle}
 
 \author{%
-Carlos Barría
+Carlos Barr\'ia
 }
 
 \setDocRef{ITTN-071}
 \setDocUpstreamLocation{\url{https://github.com/lsst-it/ittn-071}}
 
-\date{\vcsDate}
+\date{\today}
 
 % Optional: name of the document's curator
 % \setDocCurator{The Curator of this Document}
 
 \setDocAbstract{%
-following document details the procedure to update the current version of `rke` into a newer version with the objective of moving forward with the Kubernetes version installed in the designated cluster.
+following document details the procedure to update the current version of `rke` into a newer version
+with the objective of moving forward with the Kubernetes version installed in the designated cluster.
```

Comment:
The DM style for latex is to put one sentence per line (without line wrapping).

Reply:
yeah i was doing new lines manually, i think i fix it.

Reply:
The style is one long line (no `\\` escapes) to make the diff more consistent when making minor changes.